src/content/en/_index.yaml

```diff
@@ -60,7 +60,7 @@ landing_page:
         </p>
         <ul class="checkmarks">
           <li>
-            <b>Reliable</b> - Load instantly and never show the downasaur,
+            <b>Reliable</b> - Load instantly and never show the dinosaur,
```

Comment:
'Downasaur' isn't a typo. It's what we call the 8-bit t-rex on Chrome's 'Unable to connect to the Internet' message.